packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -232,6 +233,38 @@ describe("Garbage Collection Tests", () => {
 		);
 	});
 
+	it("Pending session expiry does not close container", () => {
+		let closeCalled = false;
+		const timenow = 5000; // arbitrary number
```

Comment:
You might be able to combine the two tests you wrote like this:

- `createGarbageCollector`, let the expected remaining time be X
- use `closeCalledAfterExactTicks(X)` from the previous test

This will assert that the container is _not_ closed at `X-1` but _is_ closed at `X`.

Reply:
Basically all you need to test is that the session expiry is adjusted by the `sessionExpiryTimerStarted` value, and that'll do it.